pkg/cmd/pulumi/convert.go

```diff
@@ -57,7 +57,12 @@ func loadConverterPlugin(
 	// Try and load the converter plugin for this
 	converter, err := plugin.NewConverter(ctx, name, nil)
 	if err != nil {
-		// If NewConverter returns a MissingError, we can try and install the plugin and try again.
+		// If NewConverter returns a MissingError, we can try and install the plugin if it was missing and try again,
+		// unless auto plugin installs are turned off.
+		if env.DisableAutomaticPluginAcquisition.Value() {
```

Comment:
That's pretty great. I think it will also affect the loader in tfbridge itself since it will see the ENV VAR also. Perfect. This is going to be great!